Source/Core/Core/PowerPC/JitArm64/JitArm64_LoadStorePaired.cpp

```diff
@@ -47,11 +48,11 @@ void JitArm64::psq_lXX(UGeckoInstruction inst)
   if (inst.RA || update)  // Always uses the register on update
   {
     if (indexed)
-      ADD(addr_reg, gpr.R(inst.RA), gpr.R(inst.RB));
+      ADD(addr_reg, arm_addr, gpr.R(inst.RB));
```

Comment:
Both gcc and MSVC called R(RB) before R(RA). So the order of register has changed. I doubt this is an issue here, but it can affect other (regcache) issues.

Reply:
The instruction variants that used RB were disabled by PR 10050, without affecting the issue.